client/src/app/components/Icons/IconedStatus.tsx

```diff
@@ -164,28 +157,25 @@ export const IconedStatus: React.FC<IIconedStatusProps> = ({
   };
 
   return (
-    <Flex
-      flexWrap={{ default: "nowrap" }}
-      spaceItems={{ default: "spaceItemsSm" }}
-    >
-      <FlexItem>
-        <IconWithOptionalTooltip>
-          <Icon status={status || presetProps?.status} className={className}>
-            {icon || presetProps?.icon || <UnknownIcon />}
-          </Icon>
-        </IconWithOptionalTooltip>
-      </FlexItem>
-      <FlexItem>{label || presetProps?.label}</FlexItem>
-      {(preset === "InheritedReviews" ||
+    <IconWithLabel
+      iconTooltipMessage={presetProps?.tooltipMessage}
+      icon={
+        <Icon status={status || presetProps?.status} className={className}>
+          {icon || presetProps?.icon || <UnknownIcon />}
+        </Icon>
+      }
+      label={label || presetProps?.label}
+      hasTrailingItem={
```

Comment:
Why have a flag saying to include the value of another prop instead of just including the value of the prop if it is not undefined (which would be the default)?

Reply:
@sjd78 
In general, I agree. The split was motivated by the way we currently use the component: the condition is quite long. Dedicated prop seemed more readable i.e.
```
hasTrailingItem={
    preset === "InheritedReviews" ||
    preset === "InheritedAssessments" ||
    preset === "InProgressInheritedAssessments" ||
    preset === "InProgressInheritedReviews"
  }
  trailingItem={
    <Tooltip content={getTooltipContent()}>
      <TopologyIcon />
    </Tooltip>
  }
```
If we remove the dedicated flag then a ternary operator will be required i.e.
```      
trailingItem={
    preset === "InheritedReviews" ||
    preset === "InheritedAssessments" ||
    preset === "InProgressInheritedAssessments" ||
    preset === "InProgressInheritedReviews" ? (
      <Tooltip content={getTooltipContent()}>
        <TopologyIcon />
      </Tooltip>
    ) : undefined
  }
```
If you think it's not too long then we can use single prop.

Reply:
I prefer that, or to move the trailingItem value outside of the full JSX construct

Reply:
Looking at it again quick -- you could embed a `trailingTopologyIcon` boolean to the presets and simplify the expression as well.